### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by INOPIAE**
I added some German translations.
One question to the Germans: Shall I translate "member" as "Mitglied" or shall I keep "member"

### CODE REVIEW

**Comment by Vogel612** [comment]
Member here is the technical term. I'd leave it untranslated (as should be the case with previous instances of this word)

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 1]
```diff
@@ -1210,4 +1210,94 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
...
+    <value>Rubberduck - Extract Interface</value>
+  </data>
+  <data name="DeselectAll_Button" xml:space="preserve">
+    <value>Alles abwählen</value>
+  </data>
+  <data name="SelectAll_Button" xml:space="preserve">
+    <value>Alles auswählen</value>
+  </data>
+  <data name="ExtractInteface_TitleLabel" xml:space="preserve">
+    <value>Extract Interface</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: "Extract"?? Hier vielleicht besser als: "Extrahieren"

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 2]
```diff
@@ -1210,4 +1210,94 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
...
+    <value>'{0}' hat Referenzen in mehrere Methoden.</value>
+  </data>
+  <data name="IntroduceField_Caption" xml:space="preserve">
+    <value>Rubberduck - Feld einfügen</value>
+  </data>
+  <data name="IntroduceParameter_Caption" xml:space="preserve">
+    <value>Rubberduck - Parameter einfügen</value>
+  </data>
+  <data name="IntroduceParameter_PromptIfTargetIsInterface" xml:space="preserve">
+    <value>Methode '{0}' implementiert '{1}.{2}'. Ändern der Interface Signatur? (Wird über alle Implementierungen verteilt)</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: statt "verteilt" sollte hier "angewendet" stehen. Dann wäre "über" mit "auf" besser getroffen

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 3]
```diff
@@ -1210,4 +1210,94 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
...
+  </data>
+  <data name="RefactorMenu_MoveCloserToUsage" xml:space="preserve">
+    <value>Näher an die N&amp;utzung führen</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: statt "führen" ggf. "verschieben"??

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 4]
```diff
@@ -1210,4 +1210,94 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
...
+  </data>
+  <data name="MoveCloserToUsage_Caption" xml:space="preserve">
+    <value>Rubberduck - Näher an die Nutzung führen</value>
+  </data>
+  <data name="EncapsulateField_InstructionText" xml:space="preserve">
+    <value>Spezifizierung des Eigenschaftsnamen, Parameterverfügbarkeit und 'Setter type" notwendig.</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: Mismatched quotes?

File: RetailCoder.VBE/UI/RubberduckUI.de.resx (hunk 1 above)
Comment (Hosch250) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: "ExtractInteface*"   That looks like my fault, you could change it in the base .resx as well.

**Comment by retailcoder** [comment]
Please synchronize the PR

**Comment by retailcoder** [comment]
Plenty more to translate in `Rubberduck.Inspections.InspectionsUI.resx` now; note that code inspections resource strings will eventually be moved there.

**Comment by INOPIAE** [comment]
Finished corrections

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 5]
```diff
@@ -1210,4 +1210,103 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
...
+    <value>Einrücken</value>
+  </data>
+  <data name="RefactorMenu_IntroduceField" xml:space="preserve">
+    <value>&amp;Feld einfügen</value>
+  </data>
+  <data name="PromoteVariable_InvalidSelection" xml:space="preserve">
+    <value>Die Auswahl ist keine Variable</value>
+  </data>
+  <data name="RefactorMenu_EncapsulateField" xml:space="preserve">
+    <value>G&amp;ekapseltes Feld</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: Hier geht es um den Vorgang des Kapselns. "Feld kapseln" wäre ggf. passender. 
Gleiches gilt für die Überschrift
---
It's about the action of encapsulation here. "Feld kapseln" might be a better choice. 
Same applies for the caption.

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 6]
```diff
@@ -1210,4 +1210,103 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
...
+    <value>Näher an die N&amp;utzung verschieben</value>
+  </data>
+  <data name="MoveCloserToUsage_Caption" xml:space="preserve">
+    <value>Rubberduck - Näher an die Nutzung führen</value>
+  </data>
+  <data name="EncapsulateField_InstructionText" xml:space="preserve">
+    <value>Spezifizierung des Eigenschaftsnamen, Parameterverfügbarkeit und 'Setter type' notwendig.</value>
+  </data>
+  <data name="MoveCloserToUsage_InvalidSelection" xml:space="preserve">
+    <value>Un</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: nanu? Da fehlt glaube ich etwas :smile:

**Pull request merged**